runtests.sh

```diff
@@ -456,7 +463,7 @@ fi
 
 
 # testing command to run
-cmd="python3"
+cmd="${PY_EXE}"
```

Comment:
I think the `coverage` commands could also be `${PY_EXE} -m coverage`

Reply:
done, thanks